lib/rubocop/cop/lint/format_parameter_mismatch.rb

```diff
@@ -0,0 +1,78 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Lint
+      # This lint sees if there is a mismatch between the number of
+      # expected parameters for format and what is actually passed.
+      class FormatParameterMismatch < Cop
+
+        VALID_FIELDS_REGEX = /%[bBdiouxXeEfgGacps]/
+
+        def on_send(node)
+          add_offense(node, :selector) if offending_node?(node)
+        end
+
+        private
+
+        def offending_node?(node)
+          (sprintf?(node) || format?(node)) && missing_parameters?(node)
+        end
+
+        def format_method?(name, node)
+          receiver, method_name, *args = *node
+
+          # commands have no explicit receiver
+          unless !receiver && method_name == name
```

Comment:
`unless` with `else` should always be switched to `if/else`. Way easier to comprehend for the human mind.